scannerlib/scanner.go

```diff
@@ -104,8 +104,14 @@ func validateBenchmarkConfigs(configs []*apb.BenchmarkConfig) error {
 func executeChecks(checks []configchecks.BenchmarkCheck) ([]*apb.ComplianceResult, map[string][]error) {
 	compliancePerAlternative := make(configchecks.ComplianceMap)
 	benchmarkErrors := make(map[string][]error)
+	var prvRes string
 	for _, check := range checks {
-		checkResults, err := check.Exec()
+		checkResults, res, err := check.Exec(prvRes)
+
+		if len(res) > 0 {
+			prvRes = res
```

Comment:
Can we add a test for the prvRes propagation in scanner_test.go?

Reply:
I think the test for this feature are already there:
https://github.com/google/localtoast/pull/23/files#diff-47ab99f5e07d935c740bf2ad5c3c598e5d9601daa10ca77535e235899e014ac9R862
https://github.com/google/localtoast/pull/23/files#diff-47ab99f5e07d935c740bf2ad5c3c598e5d9601daa10ca77535e235899e014ac9R890